src/wui/interactive_player.cc

```diff
@@ -605,7 +606,8 @@ void InteractivePlayer::draw_map_view(MapView* given_map_view, RenderTarget* dst
 				float opacity =
 				   f->seeing == Widelands::VisibleState::kVisible ? 1.f : kBuildhelpOpacity;
 				if (picking_starting_pos) {
-					caps = show_port_space || buildhelp() ? f->fcoords.field->nodecaps() : Widelands::CAPS_NONE;
+					caps = show_port_space || buildhelp() ? f->fcoords.field->nodecaps() :
+                                                       Widelands::CAPS_NONE;
```

Comment:
automatic formatting uses spaces only when breaking `x ? y : z`. IMHO it should use tabs as well.